### PR TITLE
zh-translation:/faq/mixer/mixer-self-monitoring.md

### PR DESCRIPTION
**Pull request by cocotyty**
ref: https://github.com/servicemesher/istio-official-translation/issues/1417

### CODE REVIEW

**Comment by cocotyty** [comment]
@googlebot I signed it!

File: content/zh/faq/mixer/mixer-self-monitoring.md [hunk 1]
```diff
@@ -1,32 +1,27 @@
...
+这提供了一些对于验证 Mixer 的性能和审计功能有用的途径：
 
-- `/metrics` provides Prometheus metrics on the Mixer process as well as gRPC
-  metrics related to API calls and metrics on adapter dispatch.
-- `/debug/pprof` provides an endpoint for profiling data in [pprof
-  format](https://golang.org/pkg/net/http/pprof/).
-- `/debug/vars` provides an endpoint exposing server metrics in JSON format.
+- `/metrics` 提供了关于 Mixer 进程相关的 Prometheus 指标 ，以及与  API 调用相关的 gRPC 的指标，和适配器分发相关的指标。
+- `/debug/pprof` 提供了用于以 [pprof 格式](https://golang.org/pkg/net/http/pprof/) 分析数据的入口。
+- `/debug/vars` 提供了用于以 JSON 格式 暴露服务指标的入口。
```
Comment (rootsongjc) [inline content/zh/faq/mixer/mixer-self-monitoring.md]: 中文之间不要加空格。

File: content/zh/faq/mixer/mixer-self-monitoring.md [hunk 2]
```diff
@@ -1,32 +1,27 @@
...
-function:
+Mixer 暴露了一个监控端口 (默认端口: `10514`)。
+这提供了一些对于验证 Mixer 的性能和审计功能有用的途径：
 
-- `/metrics` provides Prometheus metrics on the Mixer process as well as gRPC
-  metrics related to API calls and metrics on adapter dispatch.
-- `/debug/pprof` provides an endpoint for profiling data in [pprof
-  format](https://golang.org/pkg/net/http/pprof/).
-- `/debug/vars` provides an endpoint exposing server metrics in JSON format.
+- `/metrics` 提供了关于 Mixer 进程相关的 Prometheus 指标 ，以及与  API 调用相关的 gRPC 的指标，和适配器分发相关的指标。
```
Comment (rootsongjc) [inline content/zh/faq/mixer/mixer-self-monitoring.md]: API 前多了空格

File: content/zh/faq/mixer/mixer-self-monitoring.md [hunk 3]
```diff
@@ -1,32 +1,27 @@
...
+- 对于 `istio-telemetry` 服务：
 
 {{< text bash >}}
 $ kubectl -n istio-system logs -l app=telemetry -c mixer
 {{< /text >}}
 
-Mixer trace generation is controlled by command-line flags: `trace_zipkin_url`, `trace_jaeger_url`, and `trace_log_spans`. If
-any of those flag values are set, trace data will be written directly to those locations. If no tracing options are provided, Mixer
-will not generate any application-level trace information.
+Mixer 追踪信息的生成由以下命令行参数控制：`trace_zipkin_url` , `trace_jaeger_url`  和 `trace_log_spans` 。如果设置了这些参数中的任何一个，则跟踪数据将直接写入这些指定的地址。 如果没有提供追踪的选项，则 Mixer 将不会生成任何应用程序级别的追踪信息。
```
Comment (rootsongjc) [inline content/zh/faq/mixer/mixer-self-monitoring.md]: 标点符号前不要加空格。

**Pull request merged**